src/assets/scripts/client/ui/TrafficRateController.js

```diff
@@ -174,6 +185,20 @@ export default class TrafficRateController {
         this._rates = {};
         this._elements = {};
 
+
+        const airport = AirportController.airport_get();
+        this._wind = { speed: airport.wind.speed, angle: Math.round(radiansToDegrees(airport.wind.angle)) };
+
+
+        const $windDirSlider = this._buildWindSlider('wind direction', 'wind direction', this._wind.angle,
+            this._onChangeWindDirection, 10, 360, 10);
```

Comment:
see comment below, but `_onChangeWindDirection` doesn't need to be passed here since it's already on the class.